aten/src/ATen/native/native_functions.yaml

```diff
@@ -10042,18 +10042,6 @@
     CPU: slow_conv_transpose3d_cpu
     CUDA: slow_conv_transpose3d_cuda
 
-- func: slow_conv_transpose3d_backward.grad_output(Tensor grad_output, Tensor self, Tensor weight, int[3] kernel_size, int[3] stride, int[3] padding, int[3] output_padding, int[3] dilation, *, Tensor(a!) grad_input, Tensor(b!) grad_weight, Tensor(c!) grad_bias) -> (Tensor(a!), Tensor(b!), Tensor(c!))
-  python_module: nn
-  dispatch:
-    CPU: slow_conv_transpose3d_backward_out_cpu
-    CUDA: slow_conv_transpose3d_backward_out_cuda
```

Comment:
we can assume that backends aren't overriding this op, right? Since other backends (e.g. XLA) override `convolution_overrideable`, and not any of the individual ops.

Reply:
That's correct AFAIK. `convolution_overrideable` was created to be overridden for XLA. I'm sure it's still possible that someone somewhere is overriding the op, but I haven't seen anything internal or external.